Intersect (Core)/Enums/TagsPosition.cs

```diff
@@ -0,0 +1,25 @@
+namespace Intersect.Enums
+{
+	/// <summary>
+	/// Enum used for switching Npc Tags Position.
+	/// </summary>
+	public enum NpcTagPos
+	{
+		Above,
+		Under,
+		Prefix,
+		Suffix
+	}
+
+	/// <summary>
+	/// Enum used for switching Player Tags Position.
+	/// </summary>
+	public enum PlayerTagPos
```

Comment:
No need to have two enums for the same generic values.

Reply:
👍🏽  !

```
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Intersect.Enums {
	/// <summary>
	/// Enum used for switching Tags Position.
	/// </summary>
	[JsonConverter(typeof(StringEnumConverter))]
	public enum TagPosition {
		Above,
		Under,
		Prefix,
		Suffix
	}
}
```